controller/src/main/java/io/pravega/controller/store/stream/tables/serializers/EpochTransitionRecordSerializer.java

```diff
@@ -63,8 +63,8 @@ private void write00(EpochTransitionRecord epochTransitionRecord, RevisionDataOu
         revisionDataOutput.writeInt(epochTransitionRecord.getActiveEpoch());
         revisionDataOutput.writeInt(epochTransitionRecord.getNewEpoch());
         revisionDataOutput.writeLong(epochTransitionRecord.getTime());
-        revisionDataOutput.writeCollection(epochTransitionRecord.getSegmentsToSeal(), DataOutput::writeInt);
-        revisionDataOutput.writeMap(epochTransitionRecord.getNewSegmentsWithRange(), DataOutput::writeInt, this::writeValue);
+        revisionDataOutput.writeCollection(epochTransitionRecord.getSegmentsToSeal(), DataOutput::writeLong);
```

Comment:
Here and in several classes below, you are changing the formatting in a backwards incompatible way. The whole point of having write00 as opposed to wirte10 and write01 is to avoid exactly this. Please use the convention established.

Reply:
Yes, if we use the serialization to be backward compatible, we would have to handle it in the code too and not just in serialization. That will mean additional complexity in the PRs. 
The same approach is also followed with segmentstore side changes. 
quoting @andreipaduroiu from rolling txn slack channel
> There is a breaking serialization change though. If this makes it in EAP2, I'm fine with that. Otherwise we need to version it.

so there is breaking serialization change. but we are deliberately not versioning it this time for the above reason.